### ISSUE TITLE
508-defect-2 [SCREENREADER, COGNITION]: Edit buttons SHOULD have context to distinguish their purpose

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9303]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On the review screen the "Edit" links **should** indicate what they will edit.

In the screenshots below, the "Edit" link for the contact information is coded to include the context in a screen reader only span. This pattern should be applied to the first button, and the suggestion is "Edit" veteran or service member personal information.

There is also a section header missing in the first accordion, and consider adding a heading, for example "Personal information", like the one for "Contact information."

The changes should be applied to each section, Veteran, Primary, and any Secondary Caregivers.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to understand the context of edit buttons so I may accurately edit my information.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers/ in browser
2. Start screenreading device listed in Environment
3. Navigate to the review screen
4. Verify that the first edit button lacks indication about what it applies to, and the second edit button is clear

### Current code
```diff
- <button type="button" class="edit-btn primary-outline" aria-label="Edit  ">Edit</button>

+ <button type="button" class="edit-btn primary-outline" aria-label="Edit Contact information">Edit</button>
```

### Suggested code
```diff
! <button type="button" class="edit-btn primary-outline" aria-label="Edit personal information">Edit</button>

+ <button type="button" class="edit-btn primary-outline" aria-label="Edit Contact information">Edit</button>
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG Understanding consistent identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

* [Deque University - Ensure that links with the same accessible name serve a similar purpose](https://dequeuniversity.com/rules/axe/3.5/identical-links-same-purpose)

## Screenshots or Trace Logs

![Screen Shot 2020-05-19 at 4 31 18 PM](https://user-images.githubusercontent.com/57469/82375316-6987e000-99ee-11ea-857f-8859e47d64f6.png)

![Screen Shot 2020-05-19 at 4 31 36 PM](https://user-images.githubusercontent.com/57469/82375342-71478480-99ee-11ea-8561-6eb11cf8aa0e.png)

**Comment by ala-yna** [comment]
@jenstrickland - Can you validate and close? Thank you!

**Comment by jenstrickland** [comment]
Validated.
